test/common-utils.ts

```diff
@@ -0,0 +1,19 @@
+export const createKey = () => 'swr-key-' + ~~(Math.random() * 1e7)
```

Comment:
I have to create a new utils file here
otherwise the playwright will import `React Testing Library` into e2e code which would cause error